frontend/src/components/AddRecipeForm.js

```diff
@@ -90,6 +100,40 @@ const AddRecipeForm = () => {
     categoryFunction();
   }, [newCategory]);
 
+  async function getIngredienTypeFunction(newIngredientString) {
+    const newIngredientObject = JSON.parse(newIngredientString);
+    // console.log('INGREDIENMT LIST', ingredientsList);
+    // console.log('INGREDIENMT string', newIngredientString);
+    // console.log('INGREDIENMT object', newIngredientObject);
+    try {
+      const response = await axios.get(
+        `/api/ingredients/${newIngredientObject.id}`
+      );
+
+      setNewIngredient(newIngredientObject);
+      // console.log('A hozzávalókdddd', response.data);
+
+      setNewIngredientTypeList(response.data.typeList);
+      // console.log('A hozzávalók', newIngredientTypeList);
+      return response.data.typeList;
+    } catch (error) {
+      console.error();
+    }
+  }
+
+  useEffect(() => {
+    async function ingredientFunction() {
+      try {
+        const response = await axios.get(`/api/ingredients`);
+        setIngredientsList(response.data);
+        return response.data;
+      } catch (error) {
+        console.error();
```

Comment:
Itt ha már minden egy helyen van, akkor egy úttal ki lehetne íratni az errort is akár.

Reply:
nem értem mire gondolsz

Reply:
Van egy console.error() call, es egy cought error, de az error, nincs átatdva a method callnak. -> console.error(error);